.github/workflows/main.yaml

```diff
@@ -33,8 +33,10 @@ jobs:
 
       - run: pip install .
 
+      - run: pip install pytest
```

Comment:
Now that we have pyproject perhaps we can migrate to poetry to manage dev dependencies too. No need to do it as part of this PR though.